migrations/versions/0171_create_daily_sorted_letter.py

```diff
@@ -0,0 +1,30 @@
+"""
+
+Revision ID: 0171_create_daily_sorted_letter
```

Comment:
You'll need to refactor the name and Revises script name to the latest on master.